### PR TITLE
contrib/zola: new package (0.18.0)

### PR DESCRIPTION
**Pull request by jcgruenhage**

### CODE REVIEW

File: contrib/zola/template.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+sha256 = "c0e1711a68bc005c2e0ecc76a468f3459739c9e54af34850cb725d04391e19b5"
+# generates completions with host bins
+options = ["!cross"]
+
+
+def post_build(self):
+    for shell in ["bash", "fish", "zsh"]:
+        with open(self.cwd / f"zola.{shell}", "w") as outf:
+            self.do(
+                f"target/{self.profile().triplet}/release/zola",
```
Comment (jcgruenhage) [inline contrib/zola/template.py:21]: Is it preferable to have it like it is commited with a format string or joining paths like this alternative suggestion? Mostly a style question, shouldn't be a difference functionally
```suggestion
               self.cwd()
               / "target"
               / self.profile().triplet
               / "release"
               / "zola",
```

Comment (nekopsykose) [reply contrib/zola/template.py:21]: generally we seem to do fstring

Comment (nekopsykose) [reply contrib/zola/template.py:21]: and for something as short as this the multiline just makes it longer so i think that looks worse (but ofc a style preference; in other places i usually prefer multiline stuff..)

Comment (jcgruenhage) [reply contrib/zola/template.py:21]: (Well except that `self.cwd` is not a valid path in the sandboxed container, but that's just a minor detail)

Comment (nekopsykose) [reply contrib/zola/template.py:21]: for cwd there's no reason to not use ./ unless you need cwd/chroot_cwd for absolute path

Comment (nekopsykose) [reply contrib/zola/template.py:21]: (e.g. if you put it an env var for some configuration, non-absolute is weirder)

Comment (jcgruenhage) [reply contrib/zola/template.py:21]: I'll just keep it with the fstring then, shorter anyway :)

**Pull request merged**